core/app/modules/import-export/wp-cli.php

```diff
@@ -155,6 +155,17 @@ public function import( array $args, array $assoc_args ) {
 			$manifest_data = $this->get_manifest_data( $import_settings['session'] );
 			$manifest_data = $import->adapt_manifest_structure( $manifest_data );
 
+			/**
+			 * Import Export Manifest Data
+			 *
+			 * Allows 3rd parties to read and edit the kit's manifest before it is used.
+			 *
+			 * @since 3.8.0
+			 *
+			 * @param array $manifest_data The Kit's Manifest data
+			 */
+			apply_filters( 'elementor/import-export/manifest_data', $manifest_data );
```

Comment:
```suggestion
			apply_filters( 'elementor/import-export/wp-cli/manifest_data', $manifest_data );
```

Maybe you should be more specific that the filter is available only in the CLI?

Reply:
I agree. Done.